maintainers/maintainer-list.nix

```
@@ -7141,6 +7141,10 @@
     githubId = 628359;
     name = "Felix Singer";
   };
+  felixzieger = {
+    name = "Felix Zieger";
+    github = "felixzieger";
```

```suggestion
    github = "felixzieger";
    githubId = 67903933;
```
https://api.github.com/users/felixzieger

Please add email and matrix if possible as well.